### PR TITLE
adding celery main queue to pna

### PR DESCRIPTION
**Pull request by calellowitz**
@snopoke @kaapstorm @NitigyaS

### CODE REVIEW

**Comment by calellowitz** [comment]
I suspect we want to add all the missing queues but am holding off for now

File: environments/pna/app-processes.yml [hunk 1]
```diff
@@ -21,6 +21,8 @@ celery_processes:
       concurrency: 1
     async_restore_queue:
       concurrency: 1
+    celery:
+      concurrency: 1
```
Comment (esoergel) [inline environments/pna/app-processes.yml]: Is this not what line 9 accomplishes?
```
repeat_record_queue,celery,export_download_queue:
  concurrency: 1
```

**Comment by esoergel** [comment]
Not sure whether it's related, but it looks like malt tables aren't getting built on PNA either (that normally happens via these tasks:
https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/data_analytics/tasks.py

File: environments/pna/app-processes.yml (hunk 1 above)
Comment (calellowitz) [reply environments/pna/app-processes.yml]: yeah, @dannyroberts pointed that out to me. Which leaves confused how this solved the problem, or how that problem existed before, but confident that this PR should not be needed.

**Comment by calellowitz** [comment]
@esoergel guessing thats the lack of a periodic queue. i can amend this pr to include one (and eliminated the redundant main queue)

**Pull request merged**